### PR TITLE
Feature/project manipulator

### PR DESCRIPTION
**Pull request by Vyeczorny**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -7,7 +7,9 @@ let package = Package(
     name: "sbg",
     dependencies: [
         .package(url: "https://github.com/stencilproject/Stencil.git", .upToNextMinor(from: "0.13.0")),
-        .package(url: "https://github.com/tuist/xcodeproj.git", .upToNextMajor(from: "6.0.0")),
+//        .package(url: "https://github.com/tuist/xcodeproj.git", .upToNextMajor(from: "6.0.0")),
```
Comment (mateusznadolski) [inline Package.swift]: Seems like you need to cover your changes with unit test and then they will merge your PR :P

File: Tests/sbgTests/ApplicationTests.swift [hunk 2]
```diff
@@ -205,16 +206,25 @@ class ApplicationTests: QuickSpec {
                 }
 
                 context("invokes project manipulator") {
+                    // (groupPath: String, fileName: String, xcodeprojFile: String, target targetName: String){
```
Comment (mateusznadolski) [inline Tests/sbgTests/ApplicationTests.swift]: What's the point of this comment?

File: Package.swift (hunk 1 above)
Comment (Vyeczorny) [reply Package.swift]: Yep, I did it today

File: Tests/sbgTests/ApplicationTests.swift (hunk 2 above)
Comment (Vyeczorny) [reply Tests/sbgTests/ApplicationTests.swift]: I used it when I refactored the tests, now it's obsolete, I will remove it.

File: Tests/sbgTests/ApplicationTests.swift [hunk 3]
```diff
@@ -280,14 +289,24 @@ private class MockFileAdder: FileAdder {
...
+    private(set) var targetName: String!
+
     private(set) var invocationCount = 0
 
-    func addToXCodeProject(file: String, target: String) {
-        self.file = file
-        self.target = target
+    var returnedValue: Result<Void, ProjectManipulatorError>!
+
+
```
Comment (PChmiel) [inline Tests/sbgTests/ApplicationTests.swift]: redundant new lines

**Pull request merged**